package.json

```diff
@@ -209,9 +223,22 @@
 		"test-unit:update": "npm run test-unit -- --updateSnapshot",
 		"test-unit:watch": "npm run test-unit -- --watch",
 		"test-unit-php": "docker-compose run --rm wordpress_phpunit phpunit",
+		"test-unit:native": "cross-env NODE_ENV=test jest --config ./test/native/jest.config.js",
+		"test-unit:native:debug": "node --inspect node_modules/.bin/jest --runInBand --config ./test/native/jest.config.js",
 		"test-unit-php-multisite": "docker-compose run -e WP_MULTISITE=1 --rm wordpress_phpunit phpunit",
 		"playground:build": "npm run build:packages && parcel build playground/src/index.html -d playground/dist",
-		"playground:start": "concurrently \"npm run dev:packages\" \"parcel playground/src/index.html -d playground/dist\""
+		"playground:start": "concurrently \"npm run dev:packages\" \"parcel playground/src/index.html -d playground/dist\"",
+		"clean": "npm cache clean; npm clean:haste; yarn clean:jest; npm clean:metro; npm clean:react; npm clean:watchman; npm clean:node;",
+		"clean:runtime": "npm clean:haste; npm clean:react; npm clean:metro; npm clean:jest; npm clean:watchman; npm clean:babel-cache",
+		"clean:gutenberg": "npm run clean:packages && rm -rf build",
+		"clean:haste": "rm -rf /tmp/haste-map-react-native-packager-*",
+		"clean:install": "npm run clean; npm install",
+		"clean:jest": "yarn jest --clearCache --config ./test/native/jest.config.js; rm -rf $TMPDIR/jest_*",
+		"clean:metro": "rm -rf $TMPDIR/metro-cache-*;",
+		"clean:react": "rm -rf $TMPDIR/react-*",
+		"clean:node": "rm -rf node_modules",
+		"clean:watchman": "command -v watchman >/dev/null 2>&1 && watchman watch-del-all; true",
+		"clean:babel-cache": "rm -rf ./node_modules/.cache/babel-loader/*"
```

Comment:
These new clean command are not really necessary, just a helper while I was working on this.